pkgs/applications/misc/ulauncher/default.nix

```
@@ -11,22 +11,24 @@
 , intltool
 , wmctrl
 , xvfb_run
+, librsvg
 }:
 
-python27Packages.buildPythonApplication rec  {
+python36Packages.buildPythonApplication rec  {
   pname = "ulauncher";
-  version = "4.4.0.r1";
+  version = "5.6.1";
 
+  # What should we do with this?
   # Python 3 support is currently in development
   # on the dev branch and 5.x.x releases
-  disabled = ! python27Packages.isPy27;
+  # disabled = ! python36Packages.isPy27;
```

I believe you should now disable it on python2.
```suggestion
  disabled = ! python36Packages.isPy3k;
```